shared/common-adapters/text.meta.native.js

```diff
@@ -23,11 +23,11 @@ function lineClamp(lines: ?number): Object {
 
 function fontSizeToSizeStyle(fontSize: number): ?Object {
   const lineHeight = {
-    '27': 32,
-    '19': 24,
-    '17': 22,
-    '15': 20,
-    '13': 18,
+    '28': 33,
+    '20': 25,
+    '18': 23,
+    '16': 21,
+    '14': 19,
```

Comment:
Same comment as above: might be useful to add to this map rather than replacing the entries.